cherry_picker/cherry_picker.py

```diff
@@ -542,7 +553,9 @@ def abort_cherry_pick(self):
         state = self.get_state_and_verify()
         if state != WORKFLOW_STATES.BACKPORT_PAUSED:
             raise ValueError(
-                f"One can only abort a paused process. Current state: {state}. Expected state: {WORKFLOW_STATES.BACKPORT_PAUSED}"
+                "One can only abort a paused process. "
+                f"Current state: {state}. "
+                f"Expected state: {WORKFLOW_STATES.BACKPORT_PAUSED}"
```

Comment:
```suggestion
                f"One can only abort a paused process. "
                f"Current state: {state}. "
                f"Expected state: {WORKFLOW_STATES.BACKPORT_PAUSED}"
```
Any reason not to keep the `f` on all strings?
It seems more consistent and less error prone to add it.  Same applies to other strings below.

Reply:
I generally do, but I'm not sure if there are already conventions one way or the other.  The Ruff rule you mentioned makes sense in the context or lone strings without placeholders; in this case I consider the 3 strings literals (that are implicitly concatenated) as part of the same string, and I would add the same prefix(es) to all chunks.

Reply:
Added!